### ISSUE TITLE
Zoom is hidding content with screen share (POPos 21.04)

### ISSUE THREAD

**Issue by gaston-gonzalezperna** [flathub/com.vscodium.codium#57]
Whenever I share my desktop to my coworkers, it doesn't show any content like it was a Zoom Chat.
![image](https://user-images.githubusercontent.com/65687735/134740150-a4c03449-9314-4536-a54f-624fe58619f7.png)

**Comment by noonsleeper** [comment]
this problem persist with any other app? like jitsi or slack?

**Comment by vchernin** [comment]
This sounds like https://github.com/flathub/us.zoom.Zoom/issues/275

Unfortunately that issue likely won't be fixed until Zoom uses the correct freedesktop screen sharing portal, instead of the unreliable screenshoting method they seem to continue to use today.

**Comment by gaston-gonzalezperna** [comment]
> this problem persist with any other app? like jitsi or slack?

Tried with Google Meets and no issues. We dont use Slack or Jitsi to work but I can try it.



> This sounds like [flathub/us.zoom.Zoom#275](https://github.com/flathub/us.zoom.Zoom/issues/275)
> 
> Unfortunately that issue likely won't be fixed until Zoom uses the correct freedesktop screen sharing portal, instead of the unreliable screenshotting method they seem to continue to use today.
> 
> Jitsi or Slack Flatpak should work on Wayland and Xorg.

Thanks! I will open a ticket for Zoom then.
Google Meets doesn't seem to have issues, didn't tried with Jitsi or Slack because we don't use it to work.

Should I close this ticket or leave it for other people as a known issue?

**Comment by vchernin** [comment]
Yes please do open a support ticket with Zoom. Especially if you are a business customer they might actually listen. 

Zoom needs to call the freedesktop screencast xdg desktop portal. From there they'd use PipeWire to screen share. 

See https://github.com/flathub/us.zoom.Zoom/issues/22 for the very long discussion about this. The solution still is in Zoom's hands. They need to adopt the correct XDG desktop portal and PipeWire.

**Comment by noonsleeper** [comment]
> Should I close this ticket or leave it for other people as a known issue?

I'll close this because we don't have nothing in particular to fix, but don't hesitate to open a new issue if zoom fixed this and the problem persist
